src/unstract/adapters/x2text/helper.py

```diff
@@ -48,40 +64,30 @@ def process_document(
         output_file_path: Optional[str] = None,
     ) -> str:
         try:
-            input_f = open(input_file_path, "rb")
-            files = {"file": input_f}
-            response = UnstructuredHelper.make_request(
-                unstructured_adapter_config,
-                UnstructuredHelper.PROCESS,
-                files=files,
-            )
-            if response.status_code != 200:
-                logger.error(
-                    "Error in unstructured IO process document: "
-                    f"[{response.status_code}] {response.reason}"
+            response: Response
+            with open(input_file_path, "rb") as input_f:
+                mime_type = AdapterUtils.get_file_mime_type(
+                    input_file=input_file_path
                 )
-                raise AdapterError(
-                    f"{response.status_code} - {response.reason}"
+                files = {"file": (input_file_path, input_f, mime_type)}
```

Comment:
@chandrasekharan-zipstack Do you know if we are inferring file-type in the x2text-service also? Since this API call first goes to x2text-service and then to Unstructured IO, we can avoid duplicate checks.

Reply:
Seems like a duplicate comment, resolving this and answering on the first one